pkg/tsdb/azuremonitor/azuremonitor.go

```diff
@@ -30,56 +40,121 @@ func init() {
 }
 
 type Service struct {
-	PluginManager      plugins.Manager     `inject:""`
-	HTTPClientProvider httpclient.Provider `inject:""`
-	Cfg                *setting.Cfg        `inject:""`
+	PluginManager        plugins.Manager       `inject:""`
+	HTTPClientProvider   httpclient.Provider   `inject:""`
+	Cfg                  *setting.Cfg          `inject:""`
+	BackendPluginManager backendplugin.Manager `inject:""`
 }
 
-func (s *Service) Init() error {
-	return nil
+type datasourceInfo struct {
+	AppInsightsAppId             string `json:"appInsightsAppId"`
```

Comment:
I wonder if it would make sense to create a type, azureMonitorSettings, for these fields (ln 50-60)? Most of them are not used now, but they will be very soon when we start using a shared token provider for azure.

Reply:
it makes sense for me (to separate them from the rest of fields)